composer.json

```diff
@@ -18,15 +18,12 @@
     "require": {
         "composer/installers": "*",
         "silverstripe/framework": "~3.1",
-        "madmatt/simplesamlphp": "dev-ss-master"
+        "onelogin/php-saml": "v2.10.2"
```

Comment:
Updated to lock on `~2.10` instead, though somewhat irrelevant as we currently have to require people set a custom repo for that anyway, due to changes that can't (yet) be merged back into mainline... working on that separately.